ui/components/multichain/asset-picker-amount/asset-picker-modal/AssetList.tsx

```diff
@@ -67,7 +86,7 @@ export default function AssetList({
             padding={0}
             gap={0}
             margin={0}
-            key={token.symbol}
+            key={`${token.symbol}-${token.address ?? ''}`}
```

Comment:
Can a token not have an address?

Reply:
Yea, it's undefined for native assets

Reply:
We usually use the `zeroAdress()` for the address of native tokens.

Since this is just a key, I think leaving this is fine. We'll be updating this soon in the [Multichain AssetList](https://github.com/MetaMask/metamask-extension/pull/28386) so it'll be shortlived anyway